cymetric/metrics.py

```diff
@@ -229,4 +230,45 @@ def agents(series):
 del _agentsdeps, _agentsschema
 
 
+#########################
+## FCO-related metrics ##
+#########################
+
+# U Resources Mined [t] (currently implemented with a metric tonnes conversion)
+_udeps= [('Materials', ('ResourceId', 'ObjId', 'TimeCreated', 'NucId'), 'Mass'),
+         ('Transactions', ('ResourceId', ), 'Commodity')]
+
+_uschema = [('TimeCreated', ts.INT), ('FcoUMined', ts.DOUBLE)]
+
+@metric(name='FcoUMined', depends=_udeps, schema=_uschema)
+def fco_u_mined(series):
+    """FcoUMined metric returns the uranium mined at each timestep in a 
+    simulation. This is written for FCO-only databases (i.e., the U235 and 
+    U238 are given separately in the FCO simulations)."""
+    mass = pd.merge(series[0].reset_index(), series[1].reset_index(), 
+            on=['ResourceId'], how='inner').set_index(['ObjId', 
+                'TimeCreated', 'NucId'])
+    u = []
+    prods = {}
+    mass235 = {}
+    m = mass[mass['Commodity'] == 'LWR Fuel']
+    for (obj, _, nuc), value in m.iterrows():
+        prods[obj] = prods.get(obj, 0.0) + value['Mass']
+        if nuc==922350000:
+            mass235[obj] = value['Mass']
+    for obj, m235 in mass235.items():
+        x_prod = m235 / prods[obj]
+        feed = enr.feed(0.0072, x_prod, 0.0025, product=prods[obj]) / 1000
+        u.append(feed)
+    m = m.groupby(level=['ObjId', 'TimeCreated'])['Mass'].sum()
+    u = pd.Series(u, index=m.index)
+    u = u.groupby(level=['TimeCreated']).sum()
+    u.name = 'FcoUMined'
+    # sum by years (12 time steps)
+    u_year = u.groupby(pd.cut(u.index, np.arange(0, 12, 12)))['FcoUMined'].sum()
```

Comment:
Here is the gist of my error regarding this line:
https://gist.github.com/opotowsky/71d6c2ea168128fa6045